plugins/modules/grpc_nw_config.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: grpc_nw_config
+version_added: ""
+author:
+    - "Gomathi Selvi S (@GomathiselviS)"
+short_description: Fetch configuration/state data from gRPC enabled target hosts.
+description:
+    - gRPC is a high performance, open-source universal RPC framework.
+    - This module allows the user to append configs to an existing configuration in a gRPC
+      enabled devices.
+options:
+  config:
+    description:
+      - This option specifies the string which acts as a filter to restrict the portions of
+        the data to be retrieved from the target host device. If this option is not specified the entire
+        configuration or state data is returned in response provided it is supported by target host.
+  path:
+    description: path of the file that has the config in json format.
+  state:
+    description: action to be performed
+   
+requirements:
+  - grpcio
+  - protobuf
+notes:
+  - This module requires the gRPC system service be enabled on
+    the target host being managed.
+  - This module supports the use of connection=grpc.
+  - This module requires the value of 'ansible_network_os' be defined as an inventory variable.
+  - Tested against iosxrv 9k version 6.1.2.
+"""
+
+EXAMPLES = """
+- name: Merge bgp configuration data
```

Comment:
Can add example with `config` option with value provided in the task and another example of value read from file using the `file` lookup plugin.